docs/src/content/docs/guides/contribute.md

```diff
@@ -10,13 +10,13 @@ sidebar:
 
 You can contribute to this repository in many ways:
 
-🔥 Create a new challenge by following the intructions [here](/guides/create-challenge).
+🔥 Create a new challenge by following the instructions [here](/guides/create-challenge).
 
 🔥 Answer challenges and submit the results. (guide [here](/guides/resolve-challenge)).
 
-🔥 Comment on other's solutions by providing constructive and caring feedback.
+🔥 Comment on others' solutions by providing constructive and caring feedback.
```

Comment:
is that correct? `others'`?

Reply:
It is awkward.  `others's` could be right as well.  I changed the line.